### PR TITLE
openstack-crowbar: Add python backports repo for python-virtualenv

### PR DESCRIPTION
**Pull request by viccuad**
python-virtualenv is not part of the crowbar gm8+up ISO, add the
python backports repo to install it, and ensure that it always
gets removed.

### CODE REVIEW

File: scripts/jenkins/ardana/ansible/run-crowbar-tests.yml [hunk 1]
```diff
@@ -25,6 +25,16 @@
           debug:
             msg: "http://{{ ansible_host }}:9091/"
 
+        - name: Ensure temporal repo for python-virtualenv is in place
+          zypper_repository:
+            repo: "http://download.opensuse.org/repositories/devel:/languages:/python:/backports/SLE_{{ \
```
Comment (stefannica) [inline scripts/jenkins/ardana/ansible/run-crowbar-tests.yml]: For our purposes, you can simplify this to be just:
```
SLE_{{ ansible_distribution_major_version }}_SP{{ ansible_distribution_release }}
```

File: scripts/jenkins/ardana/ansible/run-crowbar-tests.yml [hunk 2]
```diff
@@ -25,6 +25,16 @@
           debug:
             msg: "http://{{ ansible_host }}:9091/"
 
+        - name: Ensure temporal repo for python-virtualenv is in place
```
Comment (stefannica) [inline scripts/jenkins/ardana/ansible/run-crowbar-tests.yml]: Nit: I would remove the `temporal` word from here and from the repository name. The fact that we're adding then removing it is enough.

File: scripts/jenkins/ardana/ansible/run-crowbar-tests.yml (hunk 1 above)
Comment (viccuad) [reply scripts/jenkins/ardana/ansible/run-crowbar-tests.yml]: I would prefer to leave it, to be honest, that way it is prepared for SLE_15 and the like.

Comment (stefannica) [reply scripts/jenkins/ardana/ansible/run-crowbar-tests.yml]: Ok, fair enough.

**Pull request merged**